### PR TITLE
bionic_prebuilt: add x86_64 support

### PR DESCRIPTION
**Pull request by s1341**
This PR adds x86_64 android support to the bionic_prebuilt package.

### CODE REVIEW

File: pkgs/os-specific/linux/bionic-prebuilt/default.nix [hunk 1]
```diff
@@ -4,23 +4,35 @@
 }:
 let
 
-  prebuilt_crt = fetchzip {
+  prebuilt_crt = if stdenv.targetPlatform.isAarch64 then (fetchzip {
     url =  "https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/+archive/98dce673ad97a9640c5d90bbb1c718e75c21e071/lib/gcc/aarch64-linux-android/4.9.x.tar.gz";
     sha256 = "sha256-LLD2OJi78sNN5NulOsJZl7Ei4F1EUYItGG6eUsKWULc=";
     stripRoot = false;
-  };
+  }) else (fetchzip {
```
Comment (Mindavi) [inline pkgs/os-specific/linux/bionic-prebuilt/default.nix:11]: Else if x86?

Comment (s1341) [reply pkgs/os-specific/linux/bionic-prebuilt/default.nix:11]: I guess we could add x86 support too. I didn't have time to track down the prebuilts for x86 and add them though. Perhaps in a separate PR?

Comment (Mindavi) [reply pkgs/os-specific/linux/bionic-prebuilt/default.nix:11]: Oops, that's not what I meant. I meant, we don't want to unconditionally do an `else`, but rather `else if x86_64`. Then it's clear that other types are unsupported.

Comment (s1341) [reply pkgs/os-specific/linux/bionic-prebuilt/default.nix:11]: Ok - that makes sense. I'll make that change

Comment (s1341) [reply pkgs/os-specific/linux/bionic-prebuilt/default.nix]: This is resolved by moving to the 'switch' idiom.

**Comment by Mindavi** [comment]
Make sure to keep contributing guidelines in mind, especially those for writing commit messages.

File: pkgs/os-specific/linux/bionic-prebuilt/default.nix [hunk 2]
```diff
@@ -39,7 +60,7 @@ let
 
 in
 stdenvNoCC.mkDerivation rec {
-  pname = "bionic-prebuilt";
+  name = "${stdenv.targetPlatform.system}-bionic-prebuilt-ndk-release-r23";
```
Comment (Mindavi) [inline pkgs/os-specific/linux/bionic-prebuilt/default.nix:63]: Please use pname here and keep the version out.

**Comment by s1341** [comment]
Woops. Sorry. I forgot the prefix. Should I rebase?

**Comment by Mindavi** [comment]
Yes, rebasing is fine

**Comment by s1341** [comment]
Should I also squash?

File: pkgs/os-specific/linux/bionic-prebuilt/default.nix [hunk 3]
```diff
@@ -39,8 +60,9 @@ let
 
 in
 stdenvNoCC.mkDerivation rec {
-  pname = "bionic-prebuilt";
+  pname = "bioinic-prebuilt";
```
Comment (Mindavi) [inline pkgs/os-specific/linux/bionic-prebuilt/default.nix:63]: Typo

**Review by Mindavi — APPROVED** [review]
LGTM otherwise

File: pkgs/os-specific/linux/bionic-prebuilt/default.nix (hunk 3 above)
Comment (s1341) [reply pkgs/os-specific/linux/bionic-prebuilt/default.nix]: EEEK!

**Comment by Mindavi** [comment]
Result of `nixpkgs-review pr 168585` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bionic</li>
  </ul>
</details>

**Comment by vcunat** [comment]
Have a look at https://github.com/NixOS/nixpkgs/pull/171268
in case you don't like the style in there or something.

**Pull request merged**